### PR TITLE
[EDU-3852] watch orch logs 

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue:
[EDU-3852](https://aziontech.atlassian.net/browse/EDU-3852)

[EDU-3852]: https://aziontech.atlassian.net/browse/EDU-3852?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: How to watch Edge Orchestrator logs
+description: Check the edge node and the edge services triggers logs.
+meta_tags: deploy, orchestrator. logs
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx:4]:
```suggestion
meta_tags: deploy, orchestrator, logs
```

File: src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+This command outputs the logs related to the edge node in specific, presenting: 
+
+- The manifest version. 
+- Communication with the server. 
+- Errors.
+
+--- 
+
+## Checking edge services logs
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx:38]:
```suggestion
## Checking edge services' logs 
```

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+import Button from '~/components/Button.astro'
+
+After the installation of Edge Orchestrator Agent, you can check the [edge node's logs](#checking-an-edge-nodes-logs) and the [edge services triggers' logs](#checking-edge-services-logs).
+
+--- 
+
+## Checking an edge node's logs
+
+1. Access your edge node.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx:18]: Como acessa o edge node?

File: src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+2. Open the terminal. 
+3. Run the following command: 
+
+```bash
+sudo vim /var/log/azion/edge-orchestrator.log 
+```
+
+:::note 
+You can choose the text editor you want, as long as it's installed on the edge node.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx:27]: How about adding an example command to install vim on an edge node? Or link to a guide on installing packages in edge nodes?

File: src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+2. Open the terminal. 
+3. Run the following command: 
+
+```bash
+sudo vim /var/log/azion/edge-services.log 
+```
+
+This command outputs the logs related to the edge node in specific, presenting the logs of the install, update, and delete triggers that were configured inside the edge services bound to the edge node.
+
+<Button href="/documentation/products/guides/deploy/advanced/run-scripts/" text="go to how to run scripts guide" variant="secondary" target="_blank"></Button>
```
Comment (bru-andrade) [inline src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx:51]:
```suggestion
<Button href="/en/documentation/products/guides/deploy/advanced/run-scripts/" text="go to how to run scripts guide" variant="secondary" target="_blank"></Button>
```

Comment (bru-andrade) [reply src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx:51]: Link this in text before maybe?

File: src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx [hunk 6]
```diff
@@ -0,0 +1,57 @@
+---
+title: How to watch Edge Orchestrator logs
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx:2]:
```suggestion
title: How to check Edge Orchestrator logs
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx:2]: monitor seems a good option too

File: src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+description: Check the edge node and the edge services triggers logs.
+meta_tags: deploy, orchestrator, logs
+namespace: docs_deploy_watch_logs
+menu_namespace: deployMenu
+permalink: /documentation/products/guides/deploy/watch-logs/
+---
+
+import Button from '~/components/Button.astro'
+
+After the installation of Edge Orchestrator Agent, you can check the [edge node's logs](#checking-an-edge-nodes-logs) and the [edge services triggers' logs](#checking-edge-services-logs).
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx:12]:
```suggestion
After the installation of **Edge Orchestrator Agent**, you can check the [edge node's logs](#checking-an-edge-nodes-logs) and the [edge services triggers' logs](#checking-edge-services-logs).
```

File: src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+- The manifest version. 
+- Communication with the server. 
+- Errors.
+
+--- 
+
+## Checking edge services' logs 
+
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx:40]:
```suggestion
```

File: src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+1. Access your edge node.
+2. Open the terminal. 
+3. Run the following command: 
+
+```bash
+sudo vim /var/log/azion/edge-services.log 
+```
+
+This command outputs the logs related to the edge node in specific, presenting the logs of the install, update, and delete triggers that were configured inside the edge services bound to the edge node.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/configure-node/watch-logs/watch-logs.mdx:49]:
```suggestion
This command outputs the logs related to the edge node in specific, presenting the logs of the install, update, and delete triggers configured inside the edge services bound to the edge node.
```

**Review by MarianaAguilera — APPROVED** [review]
with @bru-andrade 's comments

**Pull request merged**